ui/src/app/views/registration-dashboard.tsx

```diff
@@ -51,7 +51,7 @@ const styles = reactStyles({
 });
 
 function redirectToGoogleSecurity(): void {
-  let url = 'https://myaccount.google.com/security';
+  let url = 'https://myaccount.google.com/u/2/signinoptions/two-step-verification/enroll';
```

Comment:
we need to add the authed user option here so that it will go directly to that user's 2FA set up and not ask the person to sign into their google identity again

Reply:
should add `?authuser=<username>` where username is `profile.username`